### ISSUE TITLE
SLSDeep question

### ISSUE THREAD

**Issue by pkuqgg** [xmichelleshihx/AL_Skin_SAAS#1]
Hello! Sorry to bother you. Thanks for sharing the network about SLSDeep. I have trouble in testing the trained module (by myself) which is the results is lower than the paper showed. Do you have any suggestions or can you share the released module with me? It is important for me, hoping for your reply!
Thanks!

**Comment by xmichelleshihx** [comment]
Sorry, I think you have misunderstanding on my github repositories. The SLSDeep is not release by me, I just forked from other people, I think you can search SLSDeep in github and find the author. My AL_Skin_SAAS reposititory is not related to SLSDeep.

**Comment by E18301194** [comment]
@pkuqgg Hello, I am also trying to surface the accuracy of slsdeep, but I encountered some problems during debugging. What is your slsdeep pytorch version? This will help me a lot
